src/frontend/src/lib/components/convert/ConvertWizard.svelte

```diff
@@ -0,0 +1,33 @@
+<script lang="ts">
+	import type { WizardStep } from '@dfinity/gix-components';
+	import { getContext } from 'svelte';
+	import BtcConvertTokenWizard from '$btc/components/convert/BtcConvertTokenWizard.svelte';
+	import MessageBox from '$lib/components/ui/MessageBox.svelte';
+	import { CONVERT_CONTEXT_KEY, type ConvertContext } from '$lib/stores/convert.store';
+	import { i18n } from '$lib/stores/i18n.store';
+	import type { OptionAmount } from '$lib/types/send';
+	import { isNetworkIdBitcoin } from '$lib/utils/network.utils.js';
+
+	export let sendAmount: OptionAmount;
+	export let receiveAmount: number | undefined;
+	export let convertProgressStep: string;
+	export let currentStep: WizardStep | undefined;
+	export let formCancelAction: 'back' | 'close' = 'back';
+
+	const { sourceToken } = getContext<ConvertContext>(CONVERT_CONTEXT_KEY);
+</script>
+
+{#if isNetworkIdBitcoin($sourceToken?.network.id)}
+	<BtcConvertTokenWizard
+		{currentStep}
+		{formCancelAction}
+		bind:sendAmount
+		bind:receiveAmount
+		bind:convertProgressStep
+		on:icBack
+		on:icNext
+		on:icClose
+	/>
+{:else}
+	<div class="mt-6"><MessageBox>{$i18n.convert.text.unsupported_token_conversion}</MessageBox></div>
```

Comment:
Can you provide a screenshot? How does it looks like

Reply:
Sure, here it is:
<img width="579" alt="Screenshot 2024-11-20 at 11 16 34" src="https://github.com/user-attachments/assets/fe8256bf-3b33-46ef-9f25-34560332ec41">

This message can be visible only if something went wrong with the way how we use `ConvertWizard`. In other places, we render an empty slot in such cases:
<img width="602" alt="Screenshot 2024-11-20 at 11 16 49" src="https://github.com/user-attachments/assets/8747f652-bbfe-4331-b6aa-43195145cb48">